include/sys/fs/zfs.h

```diff
@@ -799,13 +799,15 @@ typedef struct pool_scan_stat {
 	uint64_t	pss_start_time;	/* scan start time */
 	uint64_t	pss_end_time;	/* scan end time */
 	uint64_t	pss_to_examine;	/* total bytes to scan */
-	uint64_t	pss_examined;	/* total examined bytes	*/
+	uint64_t	pss_examined;	/* total bytes located by scanner */
+	uint64_t	pss_issued;	/* total bytes checked by scanner */
```

Comment:
The fact that something is "required" does not really make it so.  There are always some scenarios of software update when kernel does not match the user-space.  I don't know what Linux or Illumos do about it, but for FreeBSD, thanks to such changes, we have to keep custom shim layer to provide backward compatibility for cases when user updates kernel and should still be able to use old user-level tools to manage system.